microsoft-365/business-premium/m365-campaigns-users.md

```diff
@@ -26,7 +26,7 @@ search.appverid:
 description: "Learn how these security recommendations for Microsoft 365 Business Premium affects your users and protects your data."
 ---
```

Comment:
@lhcole613 - Can you please change "affects" to "affect" and "protects" to "protect" in the metadata description? Thanks!